css-typed-om/EXPLAINER.md

```diff
@@ -0,0 +1,117 @@
+# CSS Typed OM Explained
+
+CSS Typed OM is the extensable API for the CSSOM that reduces the burden of manipulating a CSS propeerty's value via string manipulation. It does so by exposing CSS values as typed JavaScript objects which help you circumvent the browser's CSS parser.
+
+## Motivation
+
+The benefits of CSS Typed OM are as follows:
+* Allows the performant manipulation of values assigned to CSS properties. 
+* Being able to write more maintainable code:
```

Comment:
Maybe collapse this into two top level bullet points so it looks like there's more? :P

Reply:
Done